package.json

```diff
@@ -1,13 +1,11 @@
 {
   "scripts": {
     "test": "npm run spell-check",
-    "spell-check": "cspell"
+    "spell-check": "cspell",
+    "build-search-index": "pagefind"
   },
-  "@comments": [
-    "These dependencies are only for testing the site.",
-    "The site is built with Jekyll so site dependencies should be in the Gemfile."
-  ],
   "dependencies": {
-    "cspell": "5.18.5"
+    "cspell": "5.18.5",
+    "pagefind": "^0.12.0"
   }
```

Comment:
If we're moving to actually building JS stuff here we should move `cspell` into `devDependencies`.